components/RemoveFromCart.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import cookie from 'js-cookie';
+
+type Item = {
+  id: string;
+  name: string;
+  img: string;
+  amount: number;
+  price: number;
+  info: string;
+};
+
+type itemsInCart = {
+  id: string;
+  name: string;
+  img: string;
+  amount: number;
+  price: number;
+  info: string;
+};
+type Props = { item: Item; itemsInCart: itemsInCart[] };
+
+export default function RemoveFromCart(props: Props) {
+  function removeItem(id) {
+    const newCart = props.itemsInCart.filter((item) => {
+      return item.id !== id;
+    });
+    cookie.set('cart', newCart);
+
+    window.location.reload();
+    console.log(newCart);
+  }
+  return (
+    <div>
+      <button
+        data-cy={'remove-button'}
+        onClick={() => removeItem(props.item.id)}
+      >
+        Remove item from cart
+      </button>
+      <style jsx>{`
+        button {
+          align-self: center;
```

Comment:
I've seen similar `button` style blocks throughout a number of files - maybe it would make sense to make a `Button` component for most / all of them? You can still style the button with `styled-jsx` inside the component as you are used to.

Take a look at eg. https://github.com/upleveled/next-js-example-may-2020/blob/master/components/Header.js#L166